src/app/shared/utils/consts.ts

```diff
@@ -31,7 +31,7 @@ export const Consts = {
      */
     BUCKET_BACKND : new Map<string,string>(),
     BUCKET_TYPE:new Map<string,string>(),
-    BYTES_PER_CHUNK : 1024 * 1024 * 16,
+    BYTES_PER_CHUNK : 1024 * 1024 * 5,
```

Comment:
Is there any similar value to be set for Download?. Please verify and update if needed.